docs/operation/troubleshooting.rst

```diff
@@ -9,6 +9,71 @@ after a MetalK8s installation. If you do not find a solution to a problem you
 are facing, please reach out to **Scality support** or create a
 `Github issue <https://github.com/scality/metalk8s/issues>`_.
 
+.. _Troubleshooting-container:
+
+MetalK8s Troubleshooting container
+++++++++++++++++++++++++++++++++++
+
+MetalK8s ships with a dedicated container image named **metalk8s-utils**
+with the sole purpose to provide a bunch of packages used for troubleshooting
+a cluster. To startup the troubleshooting container, follow this procedure:
+
+#. On the Bootstrap, create a Deployment manifest
```

Comment:
No, this won't work. When using this image, you'll
- want it to run on a particular node
- likely want to run it in some host namespace(s), potentially with elevated privileges,...

What's below is likely not very useful, except maybe to invoke `htpasswd`, but I think `kubectl run ... --restart=Never -i ...` is a better approach there.